treeherder/config/settings.py

```diff
@@ -211,6 +211,7 @@
     Queue('calculate_durations', Exchange('default'), routing_key='calculate_durations'),
     Queue('fetch_bugs', Exchange('default'), routing_key='fetch_bugs'),
     Queue('generate_perf_alerts', Exchange('default'), routing_key='generate_perf_alerts'),
+    Queue('store_pulse_jobs', Exchange('default'), routing_key='store_pulse_jobs'),
```

Comment:
The comment from pulseload.rst saying people need to add the queue can now be removed. Could you also double check no other changes need to be made to that file?